src/server/game/Globals/ObjectMgr.cpp

```diff
@@ -3559,14 +3559,14 @@ void ObjectMgr::LoadPlayerInfo()
     {
         uint32 oldMSTime = getMSTime();
 
-        _playerXPperLevel.resize(sXpGameTable.GetTableRowCount() + 1, 0);
+        _playerXPperLevel.resize(sXpGameTable.GetTableRowCount() - 1, 0);
```

Comment:
sure, if the vector has allocated space for more rows. but you are still going out of bounds